x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/series_builder.tsx

```diff
@@ -129,53 +168,122 @@ export function SeriesBuilder({
       }),
       width: '20%',
       field: 'id',
-      render: (val: string) =>
-        reportType && indexPattern ? (
-          <ReportBreakdowns seriesId={seriesId} dataViewSeries={getDataViewSeries()} />
+      render: (seriesId: string, { series: { reportType }, seriesConfig }: BuilderItem) =>
+        reportType && seriesConfig ? (
+          <ReportBreakdowns seriesId={seriesId} dataViewSeries={seriesConfig} />
         ) : null,
     },
+    ...(multiSeries
+      ? [
+          {
+            name: i18n.translate('xpack.observability.expView.seriesBuilder.actions', {
+              defaultMessage: 'Actions',
+            }),
+            align: 'center' as const,
+            width: '10%',
+            field: 'id',
+            render: (seriesId: string, item: BuilderItem) => (
+              <SeriesActions seriesId={seriesId} editorMode={true} />
+            ),
+          },
+        ]
+      : []),
   ];
 
-  // TODO: Remove this if remain unused during multiple series view
-  // @ts-expect-error
-  const addSeries = () => {
-    if (reportType) {
-      const newSeriesId = `${
-        reportDefinitions?.['service.name'] ||
-        reportDefinitions?.['monitor.id'] ||
-        ReportViewTypes[reportType]
-      }`;
-
-      const newSeriesN: SeriesUrl = {
-        dataType,
-        time,
-        filters,
-        breakdown,
-        reportType,
-        seriesType,
-        operationType,
-        reportDefinitions,
-      };
-
-      setSeries(newSeriesId, newSeriesN);
-      removeSeries(NEW_SERIES_KEY);
-    }
+  const applySeries = () => {
+    editorItems.forEach(({ series, id: seriesId }) => {
+      const { reportType, reportDefinitions, isNew, ...restSeries } = series;
+
+      if (reportType && !isEmpty(reportDefinitions)) {
+        const reportDefId = Object.entries(reportDefinitions ?? {})[0][1];
+        const newSeriesId = `${reportDefId}-${reportType}`;
+
+        const newSeriesN: SeriesUrl = {
+          ...restSeries,
+          reportType,
+          reportDefinitions,
+        };
+
+        setSeries(newSeriesId, newSeriesN);
+        removeSeries(seriesId);
+      }
+    });
   };
 
-  const items = [{ id: seriesId }];
+  const addSeries = () => {
+    const prevSeries = allSeries?.[allSeriesIds?.[0]];
+    setSeries(
+      `${NEW_SERIES_KEY}-${editorItems.length + 1}`,
+      prevSeries
+        ? ({ isNew: true, time: prevSeries.time } as SeriesUrl)
```

Comment:
Curious: what does this time value represent? Wondering if it's a date range, but there's only a single value.